app/views/games/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Available Games</h1>
+
+<h2> 
+  <% @games.each do |game| %>
+    <% if game.white != nil && game.black == nil %>

```

Comment:
This shouldn't be necessary if you use the scope

Reply:
https://github.com/scottAF/ChestMasters/pull/15/commits/0d5d841fd174ba17e1ba210d0a2d5cad89a293d3